apps/dgst.c

```diff
@@ -414,6 +432,27 @@ int dgst_main(int argc, char **argv)
     return ret;
 }
 
+static void show_digests(const OBJ_NAME *name, void *arg)
```

Comment:
Is it worth to merge this function and `show_ciphers` into one generic function and place it in apps/lib/apps.c to avoid code duplication? (It seems only one line difference there)

Reply:
@InfoHunter I would leave it as it is for now, because it should be cherry-picked to 1.1.1.

The reason is that the situation is a bit more complicated than it seems at first sight. There  are actually three different ways to list ciphers and digests. The long term plan for master is to check whether those three implementations can be unified. But that goes beyond this pull request and would be only for 3.0.
 
(see https://github.com/openssl/openssl/issues/9893#issuecomment-531470166 and https://github.com/openssl/openssl/issues/9880#issuecomment-531478319)

Reply:
OK, that makes sense to me. Probably we could handle that part after this one is done.